pages/beta/admin/site/posts.vue

```diff
@@ -78,7 +78,14 @@
               </td>
               <td>{{ formatDate(post.created_at) }}</td>
               <td>{{ formatDate(post.last_modified) }}</td>
-              <td />
+              <td>
```

Comment:
Table header shouldn't be empty now that this column has content

Reply:
Done in 180265e